web/yarn.lock

```diff
@@ -10701,6 +10701,45 @@ __metadata:
   languageName: node
   linkType: hard
 
+"heimdalljs-fs-monitor@npm:^1.1.0":
```

Comment:
yarn lock file needs updating for the UI build. @jeffdaley could you please take a look why these changes in the lock file were triggered?

Reply:
I think it's just an alphabetizing change. Dunno how "heimdall" ever placed after "hermes" though...